src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -394,6 +414,51 @@ public static function convertWooCommerceToStorekeeperOrderStatus($wc_order_stat
         }
     }
 
+    /**
+     * @param WC_Order $databaseOrder - Order items to compare from
+     * @param $backofficeOrder - Order items to compare to
+     */
+    public function checkOrderDifference(WC_Order $databaseOrder, $backofficeOrder): bool
+    {
+        $hasDifference = false;
```

Comment:
can you test it in order only mode when the product is only in wordrpess and the fallback product is used?

Reply:
![image](https://user-images.githubusercontent.com/18332309/143547621-852b75fd-cebd-4fc5-8225-292cf3937ac1.png)
Tested on `Order only` mode and fallback is used on backoffice.